src/app/body/cart/product-card/product-card.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="product-card">
+  <div class="product-card_right">
+    <a routerLink="/products/{{product.id}}">
```

Comment:
Please, bind this attribute, use bind instead interpolation

Reply:
fixed